lms/djangoapps/onboarding/choices.py

```diff
@@ -0,0 +1,50 @@
+FUNCTIONS = (
+    ('strategy_planning', 'Strategy and planning'),
```

Comment:
let's restore the labels to how they were before, since the labels themselves are being used in Nodebb an we'd have to write a data migration to update this.
![image](https://user-images.githubusercontent.com/28558328/83020829-83ef3880-a042-11ea-8f14-9b24b74abc61.png)